package.json

```diff
@@ -46,13 +46,13 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
-    "eslint": "^4.7.2",
+    "eslint": "^4.9.0",
     "eslint-config-standard": "^10.2.1",
     "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-node": "^5.1.1",
-    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-node": "^5.2.0",
+    "eslint-plugin-promise": "^3.6.0",
     "eslint-plugin-standard": "^3.0.1",
-    "mocha": "^3.5.3",
+    "mocha": "^4.0.1",
```

Comment:
The version bumps except for `mocha` don't seem necessary.

Reply:
I just like to keep things with the latest version, given that I was updating `mocha, might as well update all the other packages.